axelrod/result_set.py

```diff
@@ -326,6 +326,26 @@ def build_ranking(self):
         return sorted(range(self.nplayers),
                       key=lambda i: -nanmedian(self.normalised_scores[i]))
 
+    def build_normalised_state_distribution(self):
```

Comment:
Needs an underscore? Seems like we're being inconsistent with the build* functions